templates/bundles/SyliusShopBundle/Checkout/SelectPayment/_choice.html.twig

```diff
@@ -1,3 +1,4 @@
+{% if not (method.gatewayConfig.gatewayName == 'tpay' and order.currencyCode != 'PLN') %}
```

Comment:
I will also after draft state try to change them for constants from php code, rather than just plain strings inside twig